procedures/foreman/branch.md.erb

```diff
@@ -62,6 +74,8 @@
 
 # Branch main code repos
 
+## Release Owner
```

Comment:
last releases tomer did a bunch of the packaging changes, so it's blury

Reply:
I'd say the packaging part should move to release engineer - i don't mind branching the repos themselves but the rest of  
"update `packages/foreman/foreman-release/foreman.gpg`, `mock/*.cfg`, `package_manifest.yaml`, `rel-eng/{releasers.conf,tito.props}` and `repoclosure/*.conf`" shouldn't be release owner. might as well move all of the branching with it since the actual branching isn't really much work.
Also the "update foreman-packaging" step should be release engineer.

Reply:
I've moved things around so it's a separate step of `Preparing build systems`. Marking as resolved.